content/en/docs/configuration/venafi.md

```diff
@@ -32,6 +32,13 @@ resources, read the [Namespaces](../../concepts/issuer/#namespaces) section.
 
 ### Creating a Venafi Cloud Issuer
 
+⚠ Since cert-manager v1.3 you will need to update your Venafi Cloud configuration to use OutagePREDICT instead of DevOpsACCELERATE.
```

Comment:
Since implies something that already happened, from is present/future (in my opinion!)

```suggestion
⚠ From cert-manager v1.3 you will need to update your Venafi Cloud configuration to use OutagePREDICT instead of DevOpsACCELERATE.
```

Reply:
Done. Although this is merging into release-next branch which will only be shown when browsing the 1.3 docs.